content/en/serverless/_index.md

```diff
@@ -34,17 +34,54 @@ further_reading:
 
 {{< vimeo 543362476 >}}
 
+ 
+<div class="alert alert-info">Make sure to check out discussions going on in the <a href="https://datadoghq.slack.com/archives/CFDPB83M4">#serverless</a> channel in the <a href="https://chat.datadoghq.com/">Datadog Slack community</a>.</div>
 
-Serverless is a concept where you write event-driven code and upload it to a cloud provider, which manages all of the underlying computational resources. [Datadog Serverless][1] brings together metrics, traces, and logs from your AWS Lambda functions running serverless applications into one view.
+[Datadog Serverless Monitoring][1] provides full visibility into all of the managed services that power your serverless applications by bringing together real-time metrics, logs and traces from your Lambda functions as well as related fully-managed APIs, queues, streams and data stores.
```

Comment:
I'm wondering if the opening should be more agnostic with regards to serverless providers. we can certainly call out that most of this page concerns AWS Lambda, but maybe a link up top here to the Azure section + maybe a mention that GCP is cool too?

Reply:
I think a quick sentence relating to Azure or Google would make sense

Reply:
resolved